packages/next/src/bin/next.ts

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env node
 
+if (process.env.NEXT_RSPACK) {
+  // silent rspack's schema check
+  process.env.RSPACK_CONFIG_VALIDATE = 'loose-silent'
+}
+
```

Comment:
ESM imports are hoisted, so this will get evaluated after the imports. Let's just move it after the imports so that ordering is more obvious.